### PR TITLE
Fix custom logo issue

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Sometimes custom logos wouldn't load. I have a hunch that this is why!  

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: app/client/components/navBar/index.tsx [hunk 1]
```diff
@@ -67,11 +67,7 @@ class NavBar extends React.Component<Props, State> {
 
     //TODO: Allow setting of region for this
     let s3_region = "https://sempo-logos.s3-ap-southeast-2.amazonaws.com";
-    let custom_url = `${s3_region}/${orgName}.${
-      deploymentName === "dev" ? "svg" : "png"
-    }`;
-
-    console.log("Custom URL is", custom_url);
+    let custom_url = `${s3_region}/${orgName}.png`;
```
Comment (michielderoos) [inline app/client/components/navBar/index.tsx:70]: The only images in that S3 bucket are png, and I'm not sure why dev would need SVGs instead.

**Comment by michielderoos** [comment]
[Look into Custom Logo Issue](https://trello.com/c/oxjX2Psm/500-look-into-custom-logo-issue)

File: app/client/components/navBar/index.tsx (hunk 1 above)
Comment (tristanhcole) [reply app/client/components/navBar/index.tsx:70]: This makes sense!!

**Review by tristanhcole — APPROVED** [review]
LGTM! Thanks @michielderoos 🙏

**Pull request merged**